flycheck.el

```diff
@@ -1110,6 +1110,22 @@ Return the path of the directory"
     (push tempdir flycheck-temporaries)
     tempdir))
 
+(defvar flycheck--forced-suffix nil
+  "List of forced filename suffixes.
+
+Dynamic variable.  If non-nil, ensure generated file names have
+one of the suffixes in this list.  Activated using
+a (force-suffix ...) clause in `flycheck-substitute-argument'.")
+
+(defun flycheck--enforce-suffix (filename)
+  "Ensure FILENAME ends with one of the suffixes in `flycheck--forced-suffix'."
+  (concat
+   filename
+   (when (and flycheck--forced-suffix
+              (-none? (lambda (suffix) (string-suffix-p suffix filename))
+                      flycheck--forced-suffix))
```

Comment:
Ah, I see now why there's a list.

Reply:
We tend to use the cl-lib functions when possible

Reply:
What is the cl-lib equivalent? I found `cl-notany`, but that's apparently in `cl-extra` which isn't currently `require`d in flycheck.

Reply:
I think just `(not (cl-some …))` :) Although here `(not (seq-some …))` would be even better, sorry :)